File: HrmsWebApiCore/WebApiCore/Models/SalaryProcess/SalaryStructureModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebApiCore.ViewModels.SalaryProcess;

namespace WebApiCore.Models.SalaryProcess
{
    public class SalaryStructureModel
    {
        public  int ID { get; set; }
        public int StructureID { get; set; }
        public int SalaryHeadID { get; set; }
        public int SalaryHeadType{ get; set; }
        public decimal Amount{ get; set; }
        public int SalaryTypeID { get; set; }
        public int BasedOnID { get; set; }
        public string CreatedDate{ get; set; }
        public int SortOrder { get; set; }
        public int CompanyID { get; set; }
        public List<SalaryStructureViewModel> AdditionModel { get; set; }
        public List<SalaryStructureViewModel> DeductionModel { get; set; }
    }
}

File: HrmsAngular/src/app/models/salary-process/salary-structure.model.ts
import { SalaryView } from "./salary-view.model";
import { SalaryStructureView } from "./salary-structure-view.model";

export class SalaryStructure {
    id:number;  
    structureID :number;
    salaryHeadID :number;
    salaryHeadType :number;
    amount :number;
    salaryTypeID :number;
    basedOnID :number;
    createdDate :string;
    sortOrder :number;
    companyID:number;
    //structureView:SalaryView[];
    additionModel:SalaryStructureView[];
    deductionModel:SalaryStructureView[];

}
